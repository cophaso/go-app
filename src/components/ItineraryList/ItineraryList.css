/* * {
  box-sizing: border-box;
}

body {
  font-family: "Open Sans", sans-serif;
  font-size: 20px;
  text-align: center;
  line-height: 1.5;
  background: #333;
  min-height: 400px;
  margin-bottom: 100px;
  clear: both;
}

ul {
  margin: 0 auto;
  text-align: center;
}

li {
  display: inline-block;
  vertical-align: top;
  margin: 10px;
  border: 5px solid #fff;
  width: 150px;
  height: 100px;
  background: #9e9d9d;
}

.name {
  text-align: center;
  display: inline-block;
} */