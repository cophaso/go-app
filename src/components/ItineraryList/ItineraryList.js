import React, { Component } from 'react';
import './ItineraryList.css'
import { Link } from 'react-router-dom'


export default class ItineraryList extends Component {
  render() {
    const { itinerary } = this.props
    return(
        <li className='itinerary-item'>
          <Link to={`/itineraries/${itinerary.id}`} className='ItineraryListItem'>
            <div className="name">{itinerary.title}</div>
          </Link>
        </li>     
    )
  }
}