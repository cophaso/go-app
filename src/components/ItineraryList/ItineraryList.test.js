import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ItineraryList from './ItineraryList';

describe(`ItineraryList component`, () => {
  const props = {
    itinerary: {
      "id": 1,
      "title": "test"
    }
  };

  it('renders the ItineraryList given props', () => {
    const wrapper = shallow(<ItineraryList {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });
})
