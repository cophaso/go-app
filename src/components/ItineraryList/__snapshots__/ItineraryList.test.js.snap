// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItineraryList component renders the ItineraryList given props 1`] = `
<li
  className="itinerary-item"
>
  <div
    className="btn-holder"
  >
    <button
      className="Note__delete"
      onClick={[Function]}
      type="button"
    >
      Delete
    </button>
  </div>
  <Link
    className="ItineraryListItem"
    to="/itineraries/1"
  >
    <div
      className="name"
    >
      test
    </div>
  </Link>
</li>
`;
